modules/services/pulseeffects.nix

```diff
@@ -9,19 +9,16 @@ let
   presetOpts = optionalString (cfg.preset != "") "--load-preset ${cfg.preset}";
 
 in {
+  imports = [
+    (mkRemovedOptionModule [ "services" "pulseeffects" "package" ]
+      "The PipeWire equivalent of Pulseeffects is called Easyeffects.")
+  ];
```

Comment:
We don't need to remove the option, only to set it to `pkgs.pulseeffects-legacy` and to remove the example.

Reply:
I figured it should be removed since now `pkgs.pipewire-pw` is an alias to `pkgs.easyeffects`, and the `pulseeffects` binary is no longer in the package - so the service would fail.

Reply:
Yes but people that have not switched to pipewire might still the need the option.

This is the pulseeffects service afterall.

Reply:
Ah, that makes sense, since they would want custom packages probably. It hadn't occured to me before. Sorry for being dense :sweat_smile: